### PR TITLE
Post wilsonville

### PR DESCRIPTION
**Pull request by celskeggs**
This includes the changes made in Wilsonville and the three-tote-auto changes from #100, because those were on the branch I was using at Wilsonville.

This needs to be code-reviewed either today or tomorrow, if possible. Issues with three-tote-auto don't matter.

Could either @amackworth, @vincent-miller, or @AidanSmith take a look at this soonish? Thanks.

### CODE REVIEW

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeToteContainer.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ protected void runAutonomous() throws InterruptedException,
         setClampOpen(true);
         waitUntilNot(Clamp.waitingForAutoCalibration);
         startSetClampHeight(0.4f);
-        collectTote(shake.get(), 5000);
+        collectTote(shake.get(), 3500);
```
Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeToteContainer.java]: Perhaps this should be tunable?

File: QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java [hunk 2]
```diff
@@ -91,7 +91,8 @@ private static void setupRollers() {
         BooleanMixing.pumpWhen(QuasarHelios.manualControl, BooleanMixing.andBooleans(rollersMode, rightTriggerPress), Rollers.rightPneumaticOverride);
 
         BooleanMixing.pumpWhen(QuasarHelios.manualControl,
-                BooleanMixing.andBooleans(rollersMode.asInvertedInput(), FloatMixing.floatIsAtLeast(leftTrigger, cutoffAuto)), QuasarHelios.autoEjector);
+                BooleanMixing.andBooleans(rollersMode.asInvertedInput(), FloatMixing.floatIsAtLeast(leftTrigger, cutoffAuto),
+                        QuasarHelios.autoHumanLoader.asInvertedInput()), QuasarHelios.autoEjector);
```
Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java]: Maybe an informative comment about the meaning of these lines would be helpful. (Took me a minute to figure out.)

**Comment by amackworth** [comment]
Just a few quick comments, but otherwise this is ready to merge. I assume this was running and tested on the robot during the event?

**Comment by celskeggs** [comment]
Yes, it was. I'll make the changes in a bit.

**Comment by celskeggs** [comment]
@amackworth I've made some changes. Would you be willing to review again?

**Comment by amackworth** [comment]
Yep, looks good. More of the complicated channel mixing stuff should probably be documented in the future, but I think this is fine for now.

**Comment by amackworth** [comment]
Oh, have these new changes been tested?

**Comment by celskeggs** [comment]
Not yet. I'd do so, but they're busy with the robot.

**Comment by amackworth** [comment]
Should we merge anyways?

**Comment by celskeggs** [comment]
Review the other PR first, that I just added: #106. They've decided to try the container grabber again in a different fashion... so the code's back. It also merges into this branch, so review & merge it first.

**Comment by amackworth** [comment]
OK, this looks good to me. Merging!

**Pull request merged**